README.rst

```diff
@@ -25,54 +40,58 @@ The main packages and tools that we build around are:
     python -m workflow.setup_project
 
     pip install -r requirements.txt
-    pip freeze > requirements.txt
+    pip install -r dev_requirements.txt
+    pip freeze > dev_requirements.txt
```

Comment:
Isn't  requirements.txt already all fixed?

Reply:
No

```
guildai>=0.7.0.post1
torch>=1.6.0
numpy>=1.18.3
ml-workflow>=0.7.4
flake8>=3.8.3
torchvision>=0.7.0
```

Reply:
Not sure if you should pip freeze to dev_req

Reply:
I agree. I think we should maybe only have a dev_requirements initially until there is a serve?

Reply:
hmm, must have looked at the wrong requirements.txt, my bad

Reply:
perhaps have it as is and discuss dev/reqs tomorrow?

Reply:
I agree. Let's discuss tomorrow. I think it's fine to change this later. Would like to merge so I can add a readthedocs project :)